pkg/util/testutil/maintenancewindow_matcher.go

```diff
@@ -1,3 +1,4 @@
+// TODO: move away from pkg, this code is only usable from tests
```

Comment:
see above

Reply:
same ticket, it affects the whole package